apps/web/src/pages/integrations/components/ChannelGroup.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Grid } from '@mantine/core';
+import { ProviderCard } from './ProviderCard';
+import { Title } from '../../../design-system';
+
+export function ChannelGroup({ title, providers }: { providers: any[]; title: string }) {
```

Comment:
Do we have an interface for the providers here?

Reply:
We do not, however, i added one on #380